.azure-pipelines/pipelines.yml

```diff
@@ -135,10 +135,17 @@ jobs:
           itemPattern: "bazel.release/envoy_binary.tar.gz"
           downloadType: single
           targetPath: $(Build.StagingDirectory)
-
+      - task: DownloadBuildArtifacts@0
+        inputs:
+          buildType: current
+          artifactName: "bazel.release.server_only.arm64"
```

Comment:
After #12124, this should be `bazel.release.arm64`.

Reply:
Thanks. Has been resolved.

Reply:
Please give some comments.